### DISCUSSION TITLE
what are the numbers in the friends list based off of?

### DISCUSSION THREAD

**Discussion by Enchiri** [vrcx-team/VRCX#806] [Q&A] [answered]
I was looking at the friends list in-app and I noticed that the left-most column is their number. What do the numbers mean?

**Comment by Provini** [comment, marked as answer]
Friend order with the highest number being your most recent friend.

The same number is represented when you click on their profile in VRCX too as "Friend no."

**Comment by TGV-Black** [comment]
Hi there

I've recently come to ask this same question

What does Friend no1 mean?

1. Because it can't be the latest friend I hung out with since I have a friend on no3 that I haven't seen in a week or two

2. but it can't be the latest one I added since my no1 a few minutes ago was a person I have had added for a while

3. It can't be who I have spent the most time with either (see statement 1)

4. I did an experiment where I had a friend on no156, whent on VRC and opened their menu a few times in different worlds and when reloading vrcx they whent down to 201 (it also changed other profiles around so my old no1 was now my no2 even though I never joined anyone or invited anyone during that session)

So what is that number actually based on?

it seems to be based on some statistic since it got my most visited and closest friends as like no1, 2 or 3

but at the same time. I can't figure out what you are basing it on for the life of me

please clarify ^^

//『TGV』Black

**Reply by Myrkie** [reply]
the numbering state in the friends list is based on of when that friend was inserted into the friends list atleast in legacy, now how that is based is unknown as VRChat is slowly migrating account friends to Redis database which has the unfortunately side effect of having seemingly randomized numbering https://github.com/vrcx-team/VRCX/issues/936
